mmv1/third_party/terraform/.teamcity/components/builds/build_parameters.kt

```diff
@@ -252,6 +252,11 @@ fun ParametrizedWithType.readOnlySettings() {
     hiddenVariable("teamcity.ui.settings.readOnly", "true", "Requires build configurations be edited via Kotlin")
 }
 
+fun ParametrizedWithType.activeBranchSettings() {
+    hiddenVariable("teamcity.activeVcsBranch.age.days", "10")
+    hiddenVariable("teamcity.activeBuildBranch.age.days", "10")
+}
```

Comment:
Could you please add a comment above this function with either a description of what these values being set means, and/or a link to some documentation for anyone who wants to learn more?